pkg/logicalbackup/logicalbackup.go

```diff
@@ -66,13 +66,61 @@ type LogicalBackuper interface {
 	Wait()
 }
 
+type BackupTables struct {
+	sync.RWMutex
+	data map[dbutils.Oid]tablebackup.TableBackuper
+}
+
+func NewBackupTables() *BackupTables {
+	return &BackupTables{sync.RWMutex{}, make(map[dbutils.Oid]tablebackup.TableBackuper)}
+}
+
+func (bt *BackupTables) GetByOIDIfExists(oid dbutils.Oid) tablebackup.TableBackuper {
```

Comment:
There's only one way to get backup table — by OID, and it's reflected in the function argument,
I'd remove ByOID from the name to have more clean code.

Reply:
Removed.